### PR TITLE
Upgrade to Flow 0.95

### PR DESCRIPTION
**Pull request by btford**
- upgrades prettier to account for new Flow syntax
- adds explicit type params in test code
- upgrades external flow type defs
- removes unused `test-utils`

### CODE REVIEW

**Comment by btford** [comment]
*extremely long sigh* looks like I need to upgrade Jest too.

File: package.json [hunk 1]
```diff
@@ -34,15 +34,15 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react": "^6.24.1",
-    "flow-bin": "0.69.0",
+    "flow-bin": "0.95.1",
```
Comment (ericsoco) [inline package.json:37]: wow that's a big jump 🎉

File: test/fixtures.js [hunk 2]
```diff
@@ -5,21 +5,21 @@ import Task, {
   drainTasksForTesting
 } from '../src/tasks';
 
-import {createStore, applyMiddleware} from 'redux';
+import {createStore, applyMiddleware, type Reducer, type Store} from 'redux';
 
-export const ECHO_TASK = taskCreator(
+export const ECHO_TASK = taskCreator<string, _, _>(
```
Comment (ericsoco) [inline test/fixtures.js:10]: You can use a literal underscore to declare a generic?

**Review by ericsoco — APPROVED** [review]

File: test/fixtures.js (hunk 2 above)
Comment (ericsoco) [reply test/fixtures.js:10]: > If you only want to specify some of the type arguments, you can use _ to let flow infer a type for you ([docs](https://flow.org/en/docs/types/generics/))

TIL. Strange they don't use `*` for this

**Comment by btford** [comment]
I'll just update babel too I guess, whatevs.

File: test/fixtures.js (hunk 2 above)
Comment (btford) [reply test/fixtures.js:10]: I think `*` is something else but  🤷‍♂️

**Comment by btford** [comment]
Landed as 38832a16155dadd3d93e19a2b5e0f54488edb7bf

**Pull request closed without merging**